.mvn/develocity.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<develocity
+  xmlns="https://www.gradle.com/develocity-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="https://www.gradle.com/develocity-maven https://www.gradle.com/schema/develocity-maven.xsd">
+  <server>
+    <url>https://develocity-staging.eclipse.org</url>
+  </server>
+  <projectId>org.eclipse.mylyn</projectId>
+  <buildScan>
+    <obfuscation>
+      <username>#{'eclipse-' + env['EF_SHORT_NAME'] + '-bot'}</username>
```

Comment:
Please replace 'eclipse-' + env['EF_SHORT_NAME'] + '-bot' with `eclipse-mylyn-bot`, this configuration only apply to gitlab-CI. I will update documentation

Reply:
I removed the line altogether as discussed on slack.